modules/core/core-api/src/main/java/com/enonic/xp/project/Projects.java

```diff
@@ -65,6 +65,11 @@ public Builder addAll( Collection<Project> projects )
             return this;
         }
 
+        public Builder addAll( Project... projects )
```

Comment:
Don't introduce varargs. They are error prone (worst, they accept null)
One can use much safer addAll(List.of(...))